modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -911,6 +911,108 @@ definition {
 		}
 	}
 
+	@description = "LPS-154872 - Verify that it's possible to filter and sort entries using metadata fields"
+	@priority = "4"
+	test CanFilterAndSortEntriesViaMetadataFields {
+		property portal.upstream = "quarantine";
+
+		task ("Create Custom Object and Custom Field (via API)") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 161889",
+				objectName = "CustomObject161889",
+				pluralLabelName = "Custom Objects 161889");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject161889");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject161889");
+		}
+
+		task ("Go to Custom Object") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 161889");
+		}
+
+		task ("Create Custom View and go to it") {
+			ObjectCustomViews.goToViewsTab();
+
+			ObjectCustomViews.addObjectViewViaUI(viewName = "Custom Views");
+
+			ObjectCustomViews.goToViewsDetails(label = "Custom Views");
+		}
+
+		var listMetaData = "Author,Create Date,Custom Field,External Reference Code,Modified Date,Status,ID";
```

Comment:
The list should be sorted alphabetically here unless there's a reason why ID is last.